keyboards/handwired/split89/rules.mk

```
@@ -0,0 +1,24 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
```

stock pro micro? and not an elite c or something?
```suggestion
BOOTLOADER = caterina
```

---

Thanks for this.  Yes, this is a stock pro micro unit.  Updated!  :)